packages/kbn-interpreter/server/server_registries.js

```diff
@@ -39,7 +39,10 @@ export const getServerRegistries = () => {
 };
 
 export const populateServerRegistries = types => {
-  if (called) throw new Error('function should only be called once per process');
+  if (called) {
+    console.log('function should only be called once per process');
+    return populatePromise;
```

Comment:
Why was this removed? Use `getServerRegistries` instead

Reply:
because of unit tests .... in unit tests we will sometimes init kibana server multiple times in the same process, where this would then throw .... 

we probably want to update tests not to do that, but i would say its out of the scope of this PR.